assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx

```diff
@@ -97,23 +103,18 @@ const content = (
         </WeeklyTextBlock>
         <WeeklyFeatureList features={[
         { title: 'UK, Europe and Rest of World', copy: '+44 (0) 330 333 6767' },
-        { title: 'Australia and New Zealand', copy: '+44 (0) 330 333 6767' },
-        { title: 'UK, Europe and Rest of World', copy: '+44 (0) 330 333 6767' },
+        { title: 'Australia and New Zealand', copy: '+61 2 8076 8599' },
+        { title: 'USA and Canada', copy: '+1 917-900-4663' },
       ]}
         />
-
       </WeeklyContentBlock>
       <WeeklyContentBlock type="white">
         <WeeklyTextBlock title="Promotion terms and conditions">
-          <p>Offer subject to availability. Guardian News and Media Limited
-          (&quot;GNM&quot;) reserves the right to withdraw this promotion
-          at any time. View full promotion terms and conditions here.
+          <p>Subscriptions available to people aged 18 and over with a valid email address. For full details of Guardian Weekly print subscription services and their terms and conditions - see <a href="https://www.theguardian.com/guardian-weekly-subscription-terms-conditions">here</a>
           </p>
         </WeeklyTextBlock>
         <WeeklyTextBlock title="Guardian Weekly terms and conditions">
-          <p>Offer subject to availability. Guardian News and Media Limited
-          (&quot;GNM&quot;) reserves the right to withdraw this promotion
-          at any time. View full promotion terms and conditions here.
+          <p>Offer subject to availability. Guardian News and Media Limited (&quot;GNM&quot;) reserves the right to withdraw this promotion at any time. For full promotion terms and conditions see <a href="https://subscribe.theguardian.com/p/WWM99X/terms?country=GB">here</a>.
```

Comment:
Beware `?country=GB`. The terms page on subscribe uses different footer content according to territory (for local contact details etc)

Reply:
oh! good point. The country code should probably carry from this page's code